src/openapi/streaming/streaming.ts

```diff
@@ -689,6 +701,17 @@ class Streaming extends MicroEmitter<EmittedEvents> {
         }
     }
 
+    /**
+     * Handles probe control messages and calls given callbacks at creation time

```

Comment:
I think part of this needs reordering and be singular - "calls callback given at"